server/tables/tables.go

```diff
@@ -341,7 +342,23 @@ func (c *CacheLog) TableName() string {
 	return "CacheLogs"
 }
 
-func ManualMigrate(db *gorm.DB) error {
+
+// Manual migration called before auto-migration.
+func PreAutoMigrate(db *gorm.DB) error {
+	m := db.Migrator()
+	if !m.HasColumn(&UserGroup{}, "status") {
```

Comment:
maybe i'm missing something, but isn't the column called membership_status?
https://github.com/buildbuddy-io/buildbuddy/blob/master/server/tables/tables.go#L178

Reply:
Yeah, I have a bad habit of sending PRs before testing them :P anyways there was a lot wrong with this PR including the "Migrator" interface actually being from GORM v2 while we are still on GORM v1.9.12 ... Updated the PR with the correct GORM usage and here's a gist with the code I used to test everything: https://gist.github.com/bduffany/419c333022e9dcadc589a33dc68adb02